src/main/java/com/epam/rd/autotasks/catalog/constant/SortEnum.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.rd.autotasks.catalog.constant;
+
+public enum SortEnum {
```

Comment:
It would be better to avoid it, because you put some logic in enum.
Please change it following way:
![image](https://user-images.githubusercontent.com/20165935/155747123-9c72868a-5890-45ec-b648-e7c694d34aa0.png)
![image](https://user-images.githubusercontent.com/20165935/155747184-07033ba4-a031-41a7-9a39-307c0b78c536.png)
![image](https://user-images.githubusercontent.com/20165935/155747283-98a0ae96-acb6-4886-8bcc-87aed4310521.png)
![image](https://user-images.githubusercontent.com/20165935/155747396-d5bd0fcd-f59f-403a-9a48-a3839c60fbe4.png)
![image](https://user-images.githubusercontent.com/20165935/155747460-4b69ae04-d932-49c9-abbc-46d3db57328d.png)